fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68686,6 +68680,304 @@ relationship: overlaps FBbt:00007053 ! lateral horn
+synonym: "larval serotonergic A3 EW1 neuron" EXACT [FlyBase:FBrf0207378]
+synonym: "larval serotonergic medial A3 neuron" EXACT []
+is_a: FBbt:00001583 ! EW1 neuron
+is_a: FBbt:00003605 ! larval serotonergic A3 neuron
+created_by: CP
+creation_date: 2018-06-26T10:39:16Z
+
+[Term]
+id: FBbt:00047336
+name: larval serotonergic A3-2 neuron
+def: "Larval serotonergic A3 neuron with its cell body located more laterally. Its contralateral branch forms a complex pattern at the ventral border of the contralateral neuropil that extends along the lateral margins. Its ipsilateral branch seems to fasciculate with the respective neurite of A3-1, though extending a contralateral neurite about mid-level in the dorso-ventral and anterior-posterior axis. This neurite extends a short way anterior along a midline tract." [FlyBase:FBrf0207378, FlyBase:FBrf0219731]
+synonym: "larval serotonergic A3 EW2 neuron" EXACT []
+synonym: "larval serotonergic lateral A3 neuron" EXACT []
+is_a: FBbt:00001584 ! EW2 neuron
+is_a: FBbt:00003605 ! larval serotonergic A3 neuron
+created_by: CP
+creation_date: 2018-06-26T10:40:14Z
+
+[Term]
+id: FBbt:00047337
+name: larval serotonergic A4-1 neuron
+def: "Larval serotonergic A4 neuron with its cell body located more medially. The axon bifurcates into two thick branches as it enters dorsally into the contralateral neuropil. These two primary branches initially diverge in opposite directions and grow along the medial two-thirds of the neuropil, forming a circular pattern, with the branches meeting at the dorsal end of the neuropil. These neurons also extend an ipsilateral neurite, that seems to fasciculate with the respective ipsilateral neurite of the A4-2 neuron, projecting anteriorly to form the majority of its branches in the next anterior segment (A3)." [FlyBase:FBrf0207378, FlyBase:FBrf0219731]
+synonym: "larval serotonergic A4 EW1 neuron" EXACT [FlyBase:FBrf0207378]
+synonym: "larval serotonergic medial A4 neuron" EXACT []
+is_a: FBbt:00001583 ! EW1 neuron
+is_a: FBbt:00003606 ! larval serotonergic A4 neuron
+created_by: CP
+creation_date: 2018-06-26T10:41:13Z
+
+[Term]
+id: FBbt:00047338
+name: larval serotonergic A4-2 neuron
+def: "Larval serotonergic A4 neuron with its cell body located more laterally. Its contralateral branch forms a complex pattern at the ventral border of the contralateral neuropil that extends along the lateral margins. Its ipsilateral branch seems to fasciculate with the respective neurite of A4-1, though extending a contralateral neurite about mid-level in the dorso-ventral and anterior-posterior axis. This neurite extends a short way anterior along a midline tract." [FlyBase:FBrf0207378, FlyBase:FBrf0219731]
+synonym: "larval serotonergic A4 EW2 neuron" EXACT []
+synonym: "larval serotonergic lateral A4 neuron" EXACT []
+is_a: FBbt:00001584 ! EW2 neuron
+is_a: FBbt:00003606 ! larval serotonergic A4 neuron
```

Comment:
Possibly add "'fasciculates with' some midline tract"?

Reply:
If so - this should be done for other cases where paper asserts taht neurite extends along some tract.